src/plugins/intel_cpu/tests/functional/custom/subgraph_tests/src/x64/moe.cpp

```diff
@@ -276,13 +277,47 @@ const std::vector<MoePatternParams> moe_params_smoke = {
                                                                     // seq_len=dynamic, hidden_size=256
         4,                                                          // topk
         8,                                                          // number_of_experts
-        512                                                         // intermediate_size
+        512,                                                        // intermediate_size
+        false
     },
     {
         {{-1, -1, 128}, {{1, 32, 128}, {1, 1, 128}, {1, 16, 128}}},  // Different seq length
         2,                                                           // topk
         4,                                                           // number_of_experts
-        256                                                          // intermediate_size
+        256,                                                         // intermediate_size
+        false
+    },
+};
+const std::vector<MoePatternParams> moe_params_full_smoke = {
+    {
+        {{-1, -1, 256}, {{2, 15, 256}, {2, 1, 256}, {3, 8, 256}}},  // data_shape,
+                                                                    // seq_len=dynamic, hidden_size=256
+        4,                                                          // topk
+        8,                                                          // number_of_experts
+        512,                                                        // intermediate_size
+        false
+    },
+    {
+        {{-1, -1, 128}, {{1, 32, 128}, {1, 1, 128}, {1, 16, 128}}},  // Different seq length
+        2,                                                           // topk
+        4,                                                           // number_of_experts
+        256,                                                         // intermediate_size
+        false
+    },
+    {
+        {{-1, -1, 256}, {{2, 15, 256}, {2, 1, 256}, {3, 8, 256}}},  // data_shape,
+                                                                    // seq_len=dynamic, hidden_size=256
+        4,                                                          // topk
+        8,                                                          // number_of_experts
+        512,                                                        // intermediate_size
+        true
+    },
+    {
+        {{-1, -1, 128}, {{1, 32, 128}, {1, 1, 128}, {1, 16, 128}}},  // Different seq length
+        2,                                                           // topk
+        4,                                                           // number_of_experts
+        256,                                                         // intermediate_size
+        true
```

Comment:
First 2 and last 2 params are identical except for with_gate_mul value, right? Maybe we can move `with_gate_mul` to `MoeTestParams` then? In this case, we will not need 2 different `moe_params_*` vectors